LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Chris Rohr
+Copyright (c) 2023 Kiwi Project
```

Comment:
Huh. I guess I never added that license copyright year update workflow to this repo...remind me tomorrow.